src/domain.jl

```diff
@@ -82,10 +82,6 @@ function affect!(integrator, f::AbstractDomainAffect{T,S,uType}) where {T,S,uTyp
         dtcache = integrator.dt
         integrator.dt *= scalefactor
```

Comment:
Why do we update `integrator.dt` instead of only a local variable `dt` (maybe that's what @gstein3m was referring to)?

Reply:
Yes, that was what I meant.